_data/kausarsayyed20/kausarsayyed20.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Kausar Salim Sayyed
+institution: AISSMS College Of Engineering, Pune 🚩 
+profile_pic: image_pp.jpeg 

```

Comment:
Incorrect file name, please do one of the following:
1. Change `image_pp.jpeg` to `kausarsayyed20.jpeg ` in the markdown file.
2. Rename `kausarsayyed20.jpeg ` to `image_pp.jpeg`.
I'd suggest you make the change in the markdown file!

Reply:
Done!!
I have change the image size to 544x544 and also renamed the file to kausarsayyed20.
Thank you!!